pandas/tests/series/test_repr.py

```diff
@@ -237,6 +237,24 @@ def test_series_repr_nat(self):
         )
         assert result == expected
 
+    @pytest.mark.parametrize(
```

Comment:
tests go in pandas/tests/io/formats/test_format.py (as that is where all of the formatting tests are)

Reply:
put after this test test_float_trim_zeros